R/AllS4.R

```diff
@@ -16,7 +16,7 @@ methods::setAs("data.table", "data.frame", function(from) {
   as.data.frame(from)
 })
 
-methods::setOldClass("IDate")
+methods::setOldClass(c("IDate", "Date"))
```

Comment:
up on L6 I see multiple calls for a similar case on the data.table/data.frame class. should we remove the one-class call there, or is there benefit from having both calls here too?

Reply:
based on the description of the test= argument I guess we should remove L6